addons/mail/models/mail_alias.py

```diff
@@ -244,6 +257,26 @@ def _get_alias_contact_description(self):
             return _('addresses linked to registered partners')
         return _('some specific addresses')
 
+    def _get_invalid_alias_bounced_body(self, message_dict):
+        """Get the body of the bounced email returned when the alias is misconfigured (ex.: error in alias_defaults).
+
+        :param message_dict: dictionary of mail values
+        """
+        body = Markup(
+            _("""<p>Dear Sender,<br /><br />
```

Comment:
Isn't it possible to extract tags from content ? At least the surrounding <p>, and maybe the "kind regards". That way content to translate is less error prone. I think we can keep the inlined <br /> as the core paragraph makes sense as a whole.

Reply:
Yes, good idea. It is changed.